dev/PushNotifications/PushNotificationManager.cpp

```diff
@@ -62,13 +64,63 @@ namespace winrt::Microsoft::Windows::PushNotifications::implementation
         }
     }
 
+    void PushNotificationManager::RegisterUnpackagedApplicationHelper(
+        const winrt::guid& remoteId,
+        _Out_ wil::unique_cotaskmem_string &unpackagedAppUserModelId)
+    {
+        wchar_t processName[1024];
+        THROW_HR_IF(ERROR_FILE_NOT_FOUND, GetModuleFileNameExW(GetCurrentProcess(), NULL, processName, sizeof(processName) / sizeof(processName[0])) == 0);
+
+        THROW_IF_FAILED(::CoInitializeEx(nullptr, COINITBASE_MULTITHREADED));
```

Comment:
See `wil::CoInitializeEx` - https://github.com/microsoft/wil/blob/492c01bb535daadf719d4445d6107aadf1e60812/include/wil/resource.h#L4347

Reply:
This was resolved, but without a change to `wil::CoInitializeEx(...)` - intentional?

Reply:
I'm taking this. I'm gonna send a new iteration.